README.md

```diff
@@ -22,4 +22,7 @@ docker-compose up
 ### Some Specifications for the App
 this app is built with the following libraries/packages:
 - [Gin Web framework](https://github.com/gin-gonic/gin/) for api routing
-- [AWS SDK](https://aws.amazon.com/sdk-for-go/) for AWS S3
\ No newline at end of file
+- [AWS SDK](https://aws.amazon.com/sdk-for-go/) for AWS S3
+
+### Future Plans
```

Comment:
nit: usually "Future Works" is the standard title for this section

Reply:
done